src/components/grid-aware/Footer/SubscriptionBlock/SubscriptionBlock.module.css

```diff
@@ -0,0 +1,77 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+/* Top Row */
+
+.subscriptionBlock {
+  background-color: var(--color-black);
+  display: flex;
+  grid-column: 1 / span 12;
+  justify-content: center;
+  padding: 35px;
+}
+
+.subscribeContainer {
+  align-items: baseline;
+  display: flex;
+}
+
+.subscribeMessage {
+  color: var(--color-white);
+  font: var(--font-subtitle);
+  margin-right: 20px;
+  width: 340px;
+}
+
+.formInput {
+  display: flex;
+  flex-direction: row;
+}
+
+.buttonWrapper {
+  margin-left: 38px;
+}
+
+.buttonWrapper > a:hover {
```

Comment:
I'm responding to the comment thread from https://github.com/ShelterTechSF/sheltertech.org/pull/89#discussion_r519107593, since GitHub was unable to preserve it because too much of the file changed since then.

Regarding your response:

> Unable to modify Button component from a parent element because the classname is hashed

You shouldn't try to modify the Button component from a parent element, period. That's the main reason I was asking us to change this. Trying to modify it breaks [encapsulation](https://en.wikipedia.org/wiki/Encapsulation_(computer_programming)).

Please just provide an alternative style that lacks the hover effect that can be controlled with props. For example, `<Button noHover>`.

Reply:
ok

Reply:
I just took it out but there is a downward shift when there is an onHover and the button appears bigger than the input bar for the mobile view

Reply:
I think you're not understanding what I'm asking you to do. I'm saying that you should not try to style the other component's elements from this one. I'm not saying that you shouldn't change the styles. If you need different styles, either add alternative styles to the other component, or just recreate the component from scratch in this one.

Styling via tag selectors on another component is very brittle because if that other component decides to change its markup, your styles will break.

Reply:
ok I see, I never passed props that way but I researched https://stackoverflow.com/questions/60816731/how-are-boolean-props-used-in-react